selfdrive/car/tesla/interface.py

```diff
@@ -35,6 +35,11 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), car_fw=None):
       ret.wheelbase = 2.959
       ret.centerToFront = ret.wheelbase * 0.5
       ret.steerRatio = 13.5
+    elif candidate == CAR.AP1_MODELS:
```

Comment:
let's merge the two since they are the same: `if candidate in [CAR.AP2_MODELS, CAR.AP1_MODELS]:`

Reply:
:+1: